main/autogen/autogen.h

```diff
@@ -83,13 +83,18 @@ struct ParsedFile {
     std::string toString(core::Context ctx);
     std::string toMsgpack(core::Context ctx, int version);
     void classlist(core::Context ctx, std::vector<std::string> &out);
-    void subclasses(core::Context ctx, std::map<std::string, std::set<std::string>> &out);
+    void subclasses(core::Context ctx, std::vector<std::string> &absolutePathsToIgnore,
+                    std::vector<std::string> &relativePathsToIgnore,
+                    std::map<std::string, std::set<std::pair<std::string, Definition::Type>>> &out);
 
 private:
     std::vector<core::NameRef> showFullName(core::Context ctx, DefinitionRef id);
     friend class MsgpackWriter;
 };
 
+void descendantsOf(std::map<std::string, std::set<std::pair<std::string, Definition::Type>>> &childMap,
```

Comment:
- `cost &` all args that can be `const &`
- `UnorderedMap`
- return result via `return`.

Reply:
[fixed](https://github.com/sorbet/sorbet/pull/1040/files#diff-fe69bef0ea7ee4e25c450060a16165f5R70)